installing/installing_alibaba/installing-alibaba-vpc.adoc

```diff
@@ -0,0 +1,71 @@
+:_content-type: ASSEMBLY
+[id="installing-alibaba-vpc"]
+= Installing a cluster on Alibaba Cloud into an existing VPC
+include::_attributes/common-attributes.adoc[]
+:context: installing-alibaba-vpc
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster into an existing Alibaba Virtual Private Cloud (VPC) on Alibaba Cloud Services. The installation program provisions the required infrastructure which can then be customized. To customize the VPC installation, modify parameters in the 'install-config.yaml' file before you install the cluster.
+
+[NOTE]
+====
+The scope of the {product-title} installation configurations is intentionally narrow. It is designed for simplicity and ensured success. You can complete many more {product-title} configuration tasks after an installation completes.
+====
+
+:FeatureName: Alibaba Cloud on {product-title}
+include::snippets/technology-preview.adoc[]
+
+[id="prerequisites_installing-alibaba-vpc"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_alibaba/preparing-to-install-on-alibaba.adoc#installation-alibaba-dns_preparing-to-install-on-alibaba[registered your domain].
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* If the cloud Resource Access Management (RAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can xref:../../installing/installing_alibaba/manually-creating-alibaba-ram.adoc#manually-creating-alibaba-ram[manually create and maintain Resource Access Management (RAM) credentials].
+
```

Comment:
Suggest to add section `About using a custom VPC` and section `Requirements for using your VPC` (For example, please refer to [Installing a cluster on GCP into an existing VPC](https://docs.openshift.com/container-platform/4.10/installing/installing_gcp/installing-gcp-vpc.html)). 

When using an existing VPC, the installation program does not create the following components: 
- VPC
- vSwitches
- Route table
- Internet NAT Gateway

Reply:
Agreed! @jianli-wei I'm adding those sections and a little more for you to review, please.